minian/cnmf.py

```diff
@@ -537,14 +596,6 @@ def get_p(y):
     output_dtypes=(float, float, float, float),
 )
 def update_temporal_cvxpy(y, g, sn, A=None, bseg=None, **kwargs):
-    """
```

Comment:
You can do that if you wish but please remember that in Git there is a hisory too

Reply:
I'd like to have it when I'm debugging the code, without having to pull up git history. Can you undo the commit that delete this?

Reply:
okay I will put the code back into the source. One remark, I will add it as line comments instead of "" """ since this is docstring comment and it will end up in RTD